lib/virtio/virtio.c

```diff
@@ -26,6 +26,20 @@ static const struct virtio_ident {
 	VIRTIO_ID_IOMEMORY, "IOMemory"}, {
 	VIRTIO_ID_SCSI, "SCSI"}, {
 	VIRTIO_ID_9P, "9P Transport"}, {
+	VIRTIO_ID_GPU, "GPU"}, {
+	VIRTIO_ID_INPUT, "Input"}, {
+	VIRTIO_ID_VSOCK, "Vsock Transport"}, {
+	VIRTIO_ID_CRYPTO, "Crypto"}, {
+	VIRTIO_ID_IOMMU, "IOMMU"}, {
+	VIRTIO_ID_MEM, "Memory"}, {
+	VIRTIO_ID_SOUND, "Sound"}, {
+	VIRTIO_ID_FS, "File System"}, {
+	VIRTIO_ID_PMEM, "Pmem"}, {
+	VIRTIO_ID_RPMB, "RPMB"}, {
+	VIRTIO_ID_SCMI, "SCMI"}, {
+	VIRTIO_ID_I2C_ADAPTER, "I2C Adapter"}, {
+	VIRTIO_ID_BT, "Bluetooth"}, {
+	VIRTIO_ID_GPIO, "GPIO" }, {
```

Comment:
Ok to add only a part of the device type. people will add more device type if needed.

Reply:
Done.